src/ert/_c_wrappers/enkf/res_config.py

```diff
@@ -685,3 +685,23 @@ def __eq__(self, other):
 
     def __ne__(self, other):
         return not self == other
+
+    def __repr__(self):
+        return f"ResConfig({str(self)})"
+
+    def __str__(self):
+        return (
+            "{\n"
+            f"SubstConfig: {self.subst_config},\n"
+            f"SiteConfig: {self.site_config},\n"
+            f"RNGConfig: {self.rng_config},\n"
+            f"AnalysisConfig: {self.analysis_config},\n"
+            f"ErtWorkflowList: {self.ert_workflow_list},\n"
+            f"HookManager: {self.hook_manager},\n"
+            f"ErtTemplates: {self.ert_templates},\n"
+            f"ECLConfig: {self.ecl_config},\n"
```

Comment:
`ECLConfig` is misspelled here. Also `RNGConfig` is gone on master so your branch is too far behind.

Reply:
fixed